packages/nextjs/src/middleware/personalize-middleware.ts

```diff
@@ -27,6 +27,7 @@ export type PersonalizeGeoData = {
  */
 export type PersonalizeMiddlewareConfig = MiddlewareBaseConfig &
   SitecoreConfig['api']['edge'] &
+  SitecoreConfig['api']['local'] &
```

Comment:
Personalize cannot function with local deployments - can we instead do a validation of edge config here and set personalizeService to null if edge config is absent?
The way we pass only the edge config into some middleware was intentional - it demostrated when middleware works and when it isn't, so we can add what was missing and just disable some middleware when in local deployment scenario.